transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -315,7 +315,7 @@ static jstring netty_epoll_native_kernelVersion(JNIEnv* env, jclass clazz) {
 }
 
 static jboolean netty_epoll_native_isSupportingSendmmsg(JNIEnv* env, jclass clazz) {
-    if (sendmmsg) {
+    if (&sendmmsg) {
```

Comment:
> When compiling on Linux, `sendmmsg` is always present, so the function is always nonnull.

The reason why we do the soft linking is to accommodate for older kernels/glibs ([added in glibc 2.14 and kernel 3.0](http://man7.org/linux/man-pages/man2/sendmmsg.2.html))where this function is not present.

IIUC what we want here is the following:

```c
return sendmmsg != NULL ? JNI_TRUE : JNI_FALSE;
```